imports/personal-access-tokens/client/personalAccessTokens.js

```diff
@@ -40,14 +43,16 @@ const showSuccessModal = (token) => {
 	}, () => {
 	});
 };
+
 Template.accountTokens.events({
 	'submit #form-tokens'(e, instance) {
 		e.preventDefault();
 		const tokenName = e.currentTarget.elements.tokenName.value.trim();
 		if (tokenName === '') {
 			return toastr.error(t('Please_fill_a_token_name'));
 		}
-		Meteor.call('personalAccessTokens:generateToken', { tokenName }, (error, token) => {
+		const bypassTwoFactor = $('#bypassTwoFactor').val() === 'true';
+		Meteor.call('personalAccessTokens:generateToken', { tokenName, bypassTwoFactor }, (error, token) => {
 			if (error) {
 				return toastr.error(t(error.error));
```

Comment:
because of this, if the user type a wrong password/totp it shows two error toasters. I don't know what to suggest to better handle this situation 🤔